Formula/jsonnet-bundler.rb

```diff
@@ -0,0 +1,24 @@
+class JsonnetBundler < Formula
+  desc "Package manager for Jsonnet"
+  homepage "https://github.com/jsonnet-bundler/jsonnet-bundler"
+  url "https://github.com/jsonnet-bundler/jsonnet-bundler.git",
+    :tag      => "v0.4.0",
+    :revision => "447344d5a038562d320a3f0dca052611ade29280"
+  license "Apache-2.0"
+
+  depends_on "go@1.12" => :build
```

Comment:
Does it not work with the latest version of go?

Reply:
> Does it not work with the latest version of go?

If you look at their repository, the `go.mod` file uses 1.12.

Reply:
Ah you're right, I'll fix it tomorrow.

Reply:
yeah, `1.12` in `go.mod` only means the minimum go version requirement. Should be fine to build with the latest `go`.

Reply:
Resolved in push.